tasks/db_tasks.py

```diff
@@ -5,6 +5,10 @@
 from tasks.helper_tasks import _echo_print
 from tasks.helper_tasks import _env_var
 
+COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
```

Comment:
Can we make use of env variables here instead of hard-coding everywhere. For example,  we can add these vars to `.env` file (Need to configured in docker-runner as well)
`
COF_FUND_ID = 47aef2f5-3fcb-4d45-acb5-f0152b5f03c4,
COF_ROUND_2_ID = c603d114-5364-4474-a0c4-c41cbf4d3bbd,
COF_ROUND_2_W3_ID = 5cf439bf-ef6f-431e-92c5-a1d90a4dd32f
`

And to access the var in the code,
`fund_id =  os.getenv("COF_FUND_ID")`

Reply:
I dont think we should be hardcoding COF IDs full stop tbf, part of another ticket to remove these things IMO